fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/auto/DruidNavigator.java

```diff
@@ -217,5 +208,13 @@ private void queryDruid(SuccessCallback successCallback, String url) {
                 },
                 url
         );
+
+        try {
+            //calling get so we wait until responses are loaded before returning and processing continues
+            responseFuture.get();
+        } catch (InterruptedException | ExecutionException e) {
```

Comment:
I see, you are right. But we should probably use the `get` with timeout version? @cdeszaq